src/draw/sw/blend/helium/lv_blend_helium.h

```diff
@@ -0,0 +1,1310 @@
+/**
+ * @file lv_blend_helium.h
+ *
+ */
+
+#ifndef LV_BLEND_HELIUM_H
+#define LV_BLEND_HELIUM_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "../../../../lv_conf_internal.h"
+
+/* detect whether helium is available based on arm compilers' standard */
+#if defined(__ARM_FEATURE_MVE) && __ARM_FEATURE_MVE
+
+#ifdef LV_DRAW_SW_HELIUM_CUSTOM_INCLUDE
+#include LV_DRAW_SW_HELIUM_CUSTOM_INCLUDE
+#endif
+
+#define USE_NATIVE_ASSEMBLY 1
```

Comment:
It should come from a compiler define.

Please also change the name to `LV_USE_NATIVE_ASSEMBLY`

Reply:
Sure, moved to lv_conf.